test-karma/client/js/client.spec.js

```diff
@@ -1569,28 +1569,37 @@ describe('GME client', function () {
             expect(clientNode.getValidSetMemberTypesDetailed('set')).to.deep.equal({'/701504349': true});
         });
 
-        it('should return all meta gme nodes synchronously',function(){
+        it('should return all meta gme nodes synchronously', function () {
             var metaNodes = client.getAllMetaNodes();
             expect(metaNodes).to.have.length(2);
             expect(!!metaNodes[0].getId).to.equal(true);
             expect(!!metaNodes[1].getId).to.equal(true);
         });
 
-        it('should check if the node is [connection]-like',function(){
+        it('should check if the node is [connection]-like', function () {
             expect(clientNode.isConnection()).to.equal(false);
         });
 
-        it('should check if the node is abstract',function(){
+        it('should check if the node is abstract', function () {
             expect(clientNode.isAbstract()).to.equal(false);
         });
 
-        it('should return the list of defined aspect names of the node',function(){
+        it('should return the list of defined aspect names of the node', function () {
             expect(clientNode.getValidAspectNames()).to.eql([]);
         });
         
-        //it('should log the textual representation of the node', function () {
-        //
-        //})
+        it('should copy the node with no parameters', function () {
+            var rootNode = client.getNode(''),
+                startChildren = rootNode.getChildrenIds().length,
+                endChildren,
+                params = {parentId: ''};
+
+            params[clientNode.getId()] = {};
+            client.copyMoreNodes(params);
+
+            endChildren = rootNode.getChildrenIds().length;
+            expect(endChildren).to.equal(startChildren + 1);
```

Comment:
I would also expect that the copy have the same registry/attributes as the original node.